README.md

```diff
@@ -4,18 +4,18 @@
 
 ## Purpose
 The existing librato modules do not seem to support tags.  This is a currently
-a minimum viable product implementation of incrementing  metric using gauges
+a minimum viable product implementation of posting a metric using gauges
 with custom tags.  (It does no aggregation and makes one
-requests per metric increment). Watch out for costs on your account if you use
-this - repeat it does on request to librato per call.
+requests per call). Watch out for costs on your account if you use
+this - repeat it does one request to librato per call.
 
 ## Usage
 
 ```js
-const increment = require('librato-simple');
+const librato = require('librato-simple');
 const config = { email: 'me@my.com, token: 'librato_token' };
-increment(config, 'mymetric.awesome', { color: 'blue', machine: 'thing' }).then(function () {
-  // we're done ...
+librato(config, 'mymetric.awesome', { color: 'blue', machine: 'thing' }).then(function () {
```

Comment:
Your code is broken:

![image](https://cloud.githubusercontent.com/assets/38154/26386202/696b3526-4088-11e7-997e-04faf8f9f9e4.png)